canonical-web-and-design/ubuntu.com
# Add cypress test for /kubernetes and /kubernetes/managed

## Done

- Add cypress test for `/kubernetes` and `/kubernetes/managed`

## QA

- Check out this feature branch
- Make sure you have the keys in the `.env.local` file
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Please do `context.skip` or `it.skip` to save time and avoid making a bunch of spam submission
- Move the `/tests/cypress/forms/forms_spec.js` file to the `/tests/cypress/integration` folder 
- Open another terminal and run `yarn run cypress open --config-file tests/cypress/cypress.json --config baseUrl=http://0.0.0.0:8001/` 
- Click `forms_spec.js` in the cypress test window
- Check the tests pass.
